### PR TITLE
add stars flickering bg animation

### LINKED ISSUE #2: Create the stars background animation
Create the stars background animation for the home page.

### PR DESCRIPTION
**Pull request by guyogev**
resolves #2

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -5,6 +5,10 @@
     <%= csrf_meta_tags %>
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+
+    <!-- BUG: do we really need those extra libs? -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.11/lodash.js"><script>
```
Comment (barstromza) [inline app/views/layouts/application.html.erb:12]: nice! we could add google fonts and icons as well

File: app/assets/javascripts/home.js [hunk 2]
```diff
@@ -1 +1,54 @@
...
+  })
+}
+
+function run() {
+  console.time('home.run')
+  console.debug('home.addStar started');
+
+  const bg = document.getElementById('star-bg');
+  const numOfStars = 100;
+      // Math.floor(Math.sqrt(bg.offsetHeight * bg.offsetWidth));
```
Comment (barstromza) [inline app/assets/javascripts/home.js]:
```
      // Math.floor(Math.sqrt(bg.offsetHeight * bg.offsetWidth));
```
is this needed here?

**Review by barstromza — COMMENTED** [review]
Nice!

File: app/assets/javascripts/home.js (hunk 2 above)
Comment (guyogev) [reply app/assets/javascripts/home.js]: Done.

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (guyogev) [reply app/views/layouts/application.html.erb:12]: done

**Pull request merged**